RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rs/databases/connect/_index.md

- [Redis client]({{< relref "/develop/clients" >}}) for your preferred programming language

For examples, see [Test client connection]({{< relref "/operate/rs/databases/connect/test-client-connectivity" >}}).

## Continue learning with Redis University

> See the [Connect to a database on Redis Software course](https://university.redis.io/course/zyxx6fdkcm5ahd) for modules.

Reviewer: ```suggestion
See the [Connect to a database on Redis Software course](https://university.redis.io/course/zyxx6fdkcm5ahd) to learn more.
```
